tests/misc/misc.exp

```diff
@@ -11,4 +11,8 @@ E.js:8:6,10: boolean
 This type is incompatible with
   E.js:4:14,19: number
 
-Found 3 errors
+F.js:2:14,17: string
```

Comment:
Hmmm...[I don't see an F.js](https://github.com/facebook/flow/tree/master/tests/misc). Did you add it and forget to add it to the commit?